foreman_ansible_core.gemspec

```diff
@@ -20,5 +20,5 @@ DESC
   s.add_development_dependency 'rubocop', '~> 0.52'
   s.add_dependency 'foreman-tasks-core', '~> 0.1'
   s.add_dependency 'foreman_remote_execution_core', '~> 1.1'
-  s.add_dependency 'net-ssh', '~> 4.0'
+  s.add_dependency 'net-ssh'
```

Comment:
Should it be `>= 4.0`?

Reply:
I don't think there's such requirement but why not, the version bundled in core for debs is probably >= 4.0, pushed new version